game/scripts/vscripts/Dialog.ts

```diff
@@ -231,9 +232,8 @@ export function playAudio(
     extraDuration?: number,
     onEnded?: () => void,
 ) {
-    const duration =
-        getSoundDuration(soundName) +
-        (extraDuration === undefined ? 0 : extraDuration);
+    const duration = getSoundDuration(soundName) + (extraDuration === undefined ? 0 : extraDuration);
+    print("Play audio", soundName, "duration:", getSoundDuration(soundName), "total-duration:", duration)
```

Comment:
Remove print.

Reply:
I actually wanted to keep this one since we are investigating the GetSoundDuration stuff